plugins/ScheduledReports/Menu.php

```diff
@@ -11,14 +11,9 @@
 
 use Piwik\Menu\MenuAdmin;
 use Piwik\Piwik;
-use Piwik\Plugins\MobileMessaging\MobileMessaging;
-use Piwik\Plugins\MobileMessaging\API as APIMobileMessaging;
 
 class Menu extends \Piwik\Plugin\Menu
 {
-    public const MOBILE_MESSAGING_TOP_MENU_TRANSLATION_KEY = 'MobileMessaging_TopMenu';
-    public const PDF_REPORTS_TOP_MENU_TRANSLATION_KEY = 'ScheduledReports_EmailReports';
```

Comment:
Looks like those translations keys were not used anywhere else. So we can remove them from the english translation files (the other files will then be updated by weblate)

Reply:
Removed